ome/ome-release
# Add logic to symlink promoted virtualbox artifacts

To test this PR:
- promote OMERO-stable, OMERO-stable-ice34 and OMERO-stable-virtualbox jobs
- run OME-merge-release-stable and check the virtualbox artifacts have been properly symlinked
